ci/scripts/e2e-source-test.sh

```diff
@@ -48,7 +48,7 @@ echo "--- e2e test w/ Rust frontend - source with kafka"
 cargo make clean-data
 cargo make ci-start ci-kafka
 ./scripts/source/prepare_ci_kafka.sh
-sqllogictest -p 4566 -d dev  './e2e_test/source/**/*.slt'
+sqllogictest -p 4566 -d dev  './e2e_test/source/*.slt'
```

Comment:
What's this change for?

Reply:
Avoid re-run test cases by this script

Reply:
What do you mean by `rerun`? 🤔

Reply:
`cdc.load.slt` and `cdc.check.slt` are in e2e_test/source/cdc/. `./e2e_test/source/**/*.slt` will also match these scripts which is not expected.

Reply:
I'm afraid someone will add a directory in the future and be confused about why they're not covered.

Reply:
Does this help this case?

https://stackoverflow.com/questions/20638040/glob-exclude-pattern

Reply:
Use a general pattern matching is not a good idea. We should arrange test cases by use case. I prefer put those existing cases to another directory.